### ISSUE TITLE
Clarify handling of true/false values for User > addresses > primaryAddress

### ISSUE THREAD

**Issue by branchedelac** [FOLIO-FSE/folio_migration_tools#265]
1. Understand current behaviour
2. Tweak current behaviour if necessary
3. Add an example value to https://github.com/FOLIO-FSE/migration_repo_template/blob/main/mapping_files/user_mapping.json

Current behaviour seems to be that lowercase true/false without quotes is translated to  "", uppercase "True"/"False" with quotes as True. Uppercase True/False without quotes is not valid json. 

Lowercase "true"/"false" seems to work: it becomes to true/"false", which FOLIO translates to true/false.

**Comment by fontanka16** [comment]
It would be great with some examples and data to try to mimic the behaviour you are experiencing.

**Comment by fontanka16** [comment]
@branchedelac  can we close this issue? Or do we need better clarifications?

**Comment by fontanka16** [comment]
@branchedelac , a reminder. Can we close this one?

**Comment by branchedelac** [comment]
@fontanka16 Have we (you) done something in the code/documentation/mapping file creator to make it clearer?

**Comment by fontanka16** [comment]
@branchedelac see the above linked commits and updated examples in migration_repo_template.

**Comment by branchedelac** [comment]
@fontanka16 I understand were not able to reproduce the following issues:

> lowercase true/false without quotes is translated to ""
nor
>  Uppercase True/False without quotes is not valid json.

The above two were what led me to believe that quoted lowercase was the only option that worked, and thus what we need to have as an example in the the migration repo user mapping file to help our colleagues map correctly.

@bltravis and I ran into this with Wellesley's users, I will see if I can find some example data/mapping file to verify.

**Comment by branchedelac** [comment]
I verified with he data migration repo template, and am seeing the (expected) behaviour that @fontanka16 describes also for lowercase unquoted true/false. If nothing has changed in the code since we ran into this back in May -- except that thing I added to make sure there is always (at least) one primary address -- I am not sure what the issue might have been for Wellesley. 

This is what I saw in May:
> Lafayette's primary address mapping values: "True", "False" Result -- both migrated as true.
> Wellesley's primary address mapping values: true, false Result -- one migrated as true, the other as "".

So I think we can close this. Thank you for writing the tests, Theodor.
